src/winuwp/runner_uwp/flutter_frameworkview.cpp

```diff
@@ -0,0 +1,155 @@
+#include "winrt/Windows.ApplicationModel.Core.h"
+#include "winrt/Windows.Foundation.h"
+#include "winrt/Windows.System.Profile.h"
+#include "winrt/Windows.System.Threading.h"
+#include "winrt/Windows.UI.Core.h"
+#include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Windows.Graphics.Display.h>
+#include <winrt/Windows.Storage.h>
+#include <winrt/Windows.UI.Popups.h>
+#include <winrt/Windows.UI.ViewManagement.Core.h>
+#include <winrt/Windows.UI.ViewManagement.h>
+
+#include <chrono>
+#include <memory>
+#include <thread>
+
+#include <flutter/flutter_view_controller.h>
+#include <flutter/flutter_windows.h>
+#include <flutter/generated_plugin_registrant.h>
+#include <flutter/plugin_registry.h>
+
+struct FlutterFrameworkView
+    : winrt::implements<
+          FlutterFrameworkView,
+          winrt::Windows::ApplicationModel::Core::IFrameworkView> {
+  // |winrt::Windows::ApplicationModel::Core::IFrameworkView|
+  void
+  Initialize(winrt::Windows::ApplicationModel::Core::CoreApplicationView const
+                 &applicationView) {
+
+    // Layout scaling must be disabled in the appinitialization phase in order
+    // to take effect correctly.
+    if (winrt::Windows::System::Profile::AnalyticsInfo::VersionInfo()
```

Comment:
Might be a small improvement to add function-local `using` statements at the top of this function (line 30). It adds a line there, but saves one here and makes this a little more readable. Same for the `TrySetDisableLayoutScaling` below.

Reply:
Actually there are a few more of these below. I think we should hold off on that cleanup and do it in a followup.

Reply:
Interesting.. stuart used to ask to remove usings ;-)  happy either way